be/src/Infrastructure/Catalog/Foundation.Catalog/Infrastructure/Products/Entities/Product.cs

```diff
@@ -31,6 +31,8 @@ public class Product : Entity
 
         public string Ean { get; set; }
 
+        public int DaysToFulfillment { get; set; }
```

Comment:
**suggestion:** It should be FulfillmentTime according to @dawiddworak88  suggestion